TwoNote/app/src/main/java/com/microsoft/device/display/samples/twonote/PenDrawView.kt

```diff
@@ -167,13 +168,24 @@ class PenDrawView : View {
         currentColor = color
     }
 
+    fun changeThickness(thickness: Int) {
+        currentThickness = thickness
+    }
+
     fun clearDrawing() {
         this.drawBitmap = Bitmap.createBitmap(width, height, Bitmap.Config.ARGB_8888)
         strokeList.clear()
 
         invalidate()
     }
 
+    fun undo() {
+        if (strokeList.isNotEmpty()) {
+            strokeList.removeAt(strokeList.lastIndex)
```

Comment:
I don't know if you noticed, but this still works if you exit the note and reload it. pretty cool